src/public/sass/application.scss

```diff
@@ -1,6 +1,89 @@
 $govuk-global-styles: true;
 
-@import 'govuk-frontend/govuk/all';
+@import "govuk-frontend/govuk/all";
```

Comment:
Not sure what the convention is, but my VScode keeps complaining when I use " " with yellow squiggly lines, and insists I use ' '.  I realise this is a rather small thing, but what do you think? (

Reply:
Yep should be single quotes 👍